Makefile

```diff
@@ -109,17 +109,31 @@ $(MOUNTPOINT)/kmk/.copied: $(shell find kmk/ -name "*.py" | xargs -0)
 	@touch $(MOUNTPOINT)/kmk/.copied
 	@sync
 
+ifdef MOUNTPOINT
+copy-compiled:
+	@echo "===> Copying compiled KMK folder"
+	@rsync -urh $(MPY_TARGET_DIR)/* $(MOUNTPOINT)/
+	@sync
+else
+copy-compiled:
+	@echo "**** MOUNTPOINT must be defined (wherever your CIRCUITPY drive is mounted) ****" && exit 1
+endif
+
 copy-kmk: $(MOUNTPOINT)/kmk/.copied
 else
 copy-kmk:
 	echo "**** MOUNTPOINT must be defined (wherever your CIRCUITPY drive is mounted) ****" && exit 1
 endif
 
-copy-board: $(MOUNTPOINT)/kb.py
-$(MOUNTPOINT)/kb.py: $(BOARD)
-	@echo "===> Copying your board to kb.py"
-	@rsync -rh $(BOARD) $@
+ifdef MOUNTPOINT
+copy-board:
+	@echo "===> Copying your board to $(MOUNTPOINT)"
+	@rsync -urh $(BOARD)/*.py $(MOUNTPOINT)/
```

Comment:
Do you want to include a check that BOARD is defined here as well?

Reply:
"Want": no. I find that whole `ifdef` target guarding clunky and inelegant. I'd be surprised if there isn't a better way to do that and meant to look into that as well at some point. As there has been interest in a cross-compilation build chain recently it would make sense to revisit the Makefile.
I mean, you're not wrong, a similar guard clause is used for keymaps. I'll do it, but wish to express my disatisfaction with that design pattern.

Reply:
I agree, nested `ifdef`s are ugly.